src/main/java/com/mitrais/chipper/temankondangan/backendapps/service/impl/EventServiceImpl.java

```diff
@@ -37,7 +38,7 @@ public EventServiceImpl(EventRepository eventRepository, UserRepository userRepo
 	@Override
 	public Event create(Long userId, CreateEventWrapper wrapper) {
 		User user = userRepository.findById(userId)
-				.orElseThrow(() -> new ResponseStatusException(HttpStatus.BAD_REQUEST, "Error: User not found!"));
+				.orElseThrow(() -> new ResourceNotFoundException("User", "id", userId));
 
 		if (wrapper.getMaximumAge() > 40 || wrapper.getMinimumAge() < 18) {
 			throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Error: Age must be between 18 and 40!");
```

Comment:
I think it should throws new BadRequestException

Reply:
I copied from the UserService before, is it changed now?

Reply:
Yep.